.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": ["react-app", "plugin:prettier/recommended"]
```

Comment:
The default number of words per line is `80 characters`. I think we should have this extended to 100 or 120.

Also, the default indent number is `4`. Don't you think this should be overridden to `2`?

Reply:
why not `80 characters` though?

Reply:
It's actually more of preference but there are some function definitions that extend more than 80 characters but are within 100-120 and are not easily breakable.

Reply:
Cool, I changed it to 100